shared/more/style-sheet.render.desktop.js

```diff
@@ -89,6 +92,81 @@ export default class Render extends Component {
   }
 }
 
+class DZ2Font extends Component {
+  render () {
+    const Space = () => <div style={{height: 20}}/>
+    return (
+      <div style={globalStyles.flexBoxColumn}>
+        <div style={globalStyles.flexBoxRow}>
+
+          {[false, true].map(darkMode => (
+            <div style={
+              {...globalStyles.flexBoxColumn,
+                flex: 1,
+                padding: 40,
+                backgroundColor: (darkMode ? globalColorsDZ2.darkBlue : globalColorsDZ2.white)}}>
+              <Text darkMode={darkMode} type='Header-Jumbo'>Header Jumbo</Text>
+              <Text darkMode={darkMode} type='Header-Jumbo'>Header Jumbo</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Header-Big'>Header Big Header Big</Text>
+              <Text darkMode={darkMode} type='Header-Big'>Header Big Header Big</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Header'>Header Header Header</Text>
+              <Text darkMode={darkMode} type='Header'>Header Header Header</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body'>Body Body</Text>
+              <Text darkMode={darkMode} type='Body'>Body Body</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body-Semibold'>Body Semibold Body Semibold</Text>
+              <Text darkMode={darkMode} type='Body-Semibold'>Body Semibold Body Semibold</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body-Small'>Body small Body Small</Text>
+              <Text darkMode={darkMode} type='Body-Small'>Body small Body Small</Text>
+              <Space/>
+
+              <div style={{alignSelf: (darkMode ? 'flex-start' : 'flex-end')}}>
+                {(darkMode ? [1.0, 0.75, 0.4] : [0.75, 0.4, 0.1]).map(opacity => (
```

Comment:
I don't understand. Body small has a default opacity. but, according to the style guide, we could do this: 
<img width="316" alt="screen shot 2016-02-15 at 5 11 37 pm" src="https://cloud.githubusercontent.com/assets/594035/13064408/485b2b0a-d407-11e5-88af-17d271485f70.png">

So to support that it makes more sense to accept an opacity. (Opacities are limited to ` 1 | 0.75 | 0.6 | 0.4` https://github.com/keybase/client/pull/2030/files#diff-b0d6e1e0ad4551c3f5a944be3d0ac544R11)